src/test/java/org/mockitousage/basicapi/MocksSerializationTest.java

```diff
@@ -394,6 +395,13 @@ public void private_constructor_currently_not_supported_at_the_moment_at_deseria
 
     @Test
     public void BUG_ISSUE_399_try_some_mocks_with_current_answers() throws Exception {
+        // for some reason this test fails with NullPointerException java 6, but work on java 7+
+        // java.lang.NullPointerException
+        // at java.util.LinkedList.writeObject(LinkedList.java:942)
+        // ...
+        // at org.mockitoutil.SimpleSerializationUtil.serializeMock(SimpleSerializationUtil.java:34)
```

Comment:
We should extract an issue for this to investigate (and potentially fix for Java 6)